dist/common/scripts/scylla_util.py

```diff
@@ -173,6 +174,17 @@ def _non_root_nvmes(self):
         nvmes_present = list(filter(nvme_re.match, os.listdir("/dev")))
         return {self.ROOT: root_devs, self.EPHEMERAL: [x for x in nvmes_present if not self.is_in_root_devs(x, root_devs)]}
 
+    def _non_root_disks(self):
+        """get list of disks from os, filter away if one of them is root"""
+        disk_re = re.compile(r"sd[b-z]+$")
+
+        root_dev_candidates = [x for x in psutil.disk_partitions() if x.mountpoint == "/"]
+
+        root_devs = [x.device for x in root_dev_candidates]
+
+        disks_present = list(filter(disk_re.match, os.listdir("/dev")))
```

Comment:
I need to rebase a glob.glob patch to have listdir done in a faster and cleaner way
see https://github.com/scylladb/scylla/pull/8779/commits/ff91a44b7358ec0324341be6403065534c462738
can you rewrite to glob @syuu1228 ?

Reply:
@tarzanek rewrited.